wwudora.module

```diff
@@ -14,3 +14,31 @@ function wwudora_islandora_oai_get_xsl_files() {
     "$module_path/xml/wwu_mods_to_dc.xsl" => 'wwu_mods_to_dc.xsl',
   );
 }
+
+/**
+ * Implements hook_islandora_compound_object_children_added_to_parent().
+ */
+function wwudora_islandora_compound_object_children_added_to_parent($children, $parent_pids) {
+  $parent = islandora_object_load(current($parent_pids));
```

Comment:
There could conceivably be more than one parent here though right? I know it's not possible by default in the UI, but other modules could alter the form or call [this function](https://github.com/Islandora/islandora_solution_pack_compound/blob/7.x/includes/manage.form.inc#L431) directly with more than one parent.

Reply:
Nevermind, this makes sense this way. Otherwise the child XACML would just get overwritten by its parents. We can keep it like this, but should probably add a comment explaining why